webpack/ForemanTasks/Components/TaskDetails/Components/Task.js

```diff
@@ -37,6 +38,8 @@ class Task extends Component {
       showForceUnlockModal,
       toggleUnlockModal,
       toggleForceUnlockModal,
+      cancelTask,
+      action,
```

Comment:
action needs to have a `propType`

Reply:
they have a prop type in [Tasks Info](https://github.com/theforeman/foreman-tasks/blob/2c97ec1653c756cd580e4bedb5c4f040608cc63e/webpack/ForemanTasks/Components/TaskDetails/Components/TaskInfo.js#L217)  [Imported here](https://github.com/theforeman/foreman-tasks/blob/2c97ec1653c756cd580e4bedb5c4f040608cc63e/webpack/ForemanTasks/Components/TaskDetails/Components/Task.js#L168)
Should I move it from taskInfo to Task?

Reply:
Thanks! I missed that, looks good in this case :+1: